static/README.md

```diff
@@ -0,0 +1,94 @@
+# Raster Foundry
+
+A complete, yet simple, starter for Angular using Webpack.
```

Comment:
yea, they're from my initial setup. I was gonna re-write them before I merged, but it can wait until your PR